docs/markdown/Subprojects.md

```diff
@@ -84,12 +84,13 @@ return variables as strings.
 ### Build options in subproject
 
 All Meson features of the subproject, such as project options keep
-working and can be set in the master project. There are a few
-limitations, the most important being that global compiler arguments
-must be set in the main project before calling subproject. Subprojects
-must not set global arguments because there is no way to do that
-reliably over multiple subprojects. To check whether you are running
-as a subproject, use the `is_subproject` function.
+working and [can be set in the master
+project](https://mesonbuild.com/Builtin-options.html#specifying-options-per-subproject).
```

Comment:
```suggestion
project](Builtin-options.md#specifying-options-per-subproject).
```

The markdown files don't "know" that they are converted to HTML, and they definitely shouldn't hardcode the domain name. Elsewhere in the docs, we always do inter-doc links by referring to the markdown files.